website/templates/public/pages/faq.mako

```diff
@@ -118,7 +118,7 @@
 <p>If the project is private, claim the account, and then follow the steps above.</p>
 
 <h3>How do I make folders to organize the files in my project?</h3>
-<p>Components act like folders. Thus, to organize your files into folders, create components. From the project overview page, click “create component” on the right half of the page.</p>
+<p>You can create a folder or a component to help organize your files. To create a folder, navigate to the files tab of your project. Find the storage add-on you'd like to put a folder in and click the "+" sign next to its name. Components also act like folders, but can be used to control privacy settings independently of the whole project. To create a component, navigate to the project overview page and click “create component” on the right half of the page.</p>
```

Comment:
We used to get a lot of questions about how to make folders - because the
connection between folders and components wasn't obvious for a lot of
users.

Now that there are folders, there essentially two ways to organize
research: use folders (behavior is generally what people expect) or use
components (added functionality - fine grained privacy and sharing, GUIDs,
etc.). I updated the FAQ to explain those options.

Alternative 1: Remove this FAQ and add one explaining how folders and
components are different
Alternative 2: Remove this FAQ altogether, update language on getting
started page (needs to be done anyway) to reflect addition of folders, and
see if this continues to be a question

I vote for alternative 2

On Fri, May 8, 2015 at 10:17 AM, Steven Loria <notifications@github.com>
wrote:

> In website/templates/public/pages/faq.mako
> <https://github.com/CenterForOpenScience/osf.io/pull/2662#discussion_r29942942>
> :
>
> > @@ -118,7 +118,7 @@
> >  <p>If the project is private, claim the account, and then follow the steps above.</p>
> >
> >  <h3>How do I make folders to organize the files in my project?</h3>
> > -<p>Components act like folders. Thus, to organize your files into folders, create components. From the project overview page, click “create component” on the right half of the page.</p>
> > +<p>You can create a folder or a component to help organize your files. To create a folder, navigate to the files tab of your project. Find the storage add-on you'd like to put a folder in and click the "+" sign next to its name. Components also act like folders, but can be used to control privacy settings independently of the whole project. To create a component, navigate to the project overview page and click “create component” on the right half of the page.</p>
>
> The question is "How do I make folders to organize the files in my
> project?"
>
> I'm also unsure if this needs to be in the FAQ.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CenterForOpenScience/osf.io/pull/2662/files#r29942942>
> .
>